enterprise/frontend/src/embedding-sdk/README.md

```diff
@@ -110,11 +110,11 @@ const metabaseConfig = {
 ### JWT Authentication
 
 1. Go to Admin settings > Authentication > JWT
```

Comment:
for the next time, when you see that a file needs to be formatted, can you do that in a separate PR?
This pr has been open for 3 weeks, we're lucky we didn't get conflicts

Reply:
crap, I thought the diff was large because I wrote a decent amount... 🙃

Reply:
nice catch. I'll fix that before merging